.rubocop.yml

```diff
@@ -75,3 +75,6 @@ Style/RegexpLiteral:
 
 Style/SlicingWithRange:
   Enabled: false
+
+Metrics/ParameterLists:
```

Comment:
please remove it here, and add `# rubucop:disable Metrics/ParameterLists` before the method and after `# rubucop:enable Metrics/ParameterLists`